### ISSUE TITLE
Management of Proxy Manager not available after update to 0.2.0

### ISSUE THREAD

**Issue by reharmsen** [hassio-addons/addon-nginx-proxy-manager#17]
# Problem/Motivation
After updating the nginx-proxy-manager addon to 0.2.0 I am no longer able to access the Proxy management web interface via port 81.

## Expected behavior
I expected the management web interface

## Actual behavior
Timeout on connection

## Steps to reproduce
Start the ngins proxy manager addon and browse to the webinterface
eg. http://vm-hassos01:81/

## Proposed changes
...

Possible relevant information:
 Add-on version: 0.2.0
 System: HassOS 2.12  (amd64 / qemux86-64)
 Home Assistant version: 0.93.1
 Supervisor version: 164

Port 80 and 443 are working as I am able to use my home assistant setup via the proxy.

**Comment by reharmsen** [comment]
Perpaps this error in the logs is relevant:`... keep getting this about every second

```
5/18/2019] [11:28:51 PM] [Global   ] › ✖  error     select * from `migrations_lock` - ER_UNKNOWN_STORAGE_ENGINE: Unknown storage engine 'InnoDB'
```

**Comment by frenck** [comment]
Fixed in v0.2.1, sorry about that.
